packages/sdk/contractkit/src/wrappers/LockedGold.ts

```diff
@@ -316,11 +316,25 @@ export class LockedGoldWrapper extends BaseWrapperForGoverning<LockedGold> {
    * @param epochNumber The epoch to retrieve AccountSlashed at.
    */
   async getAccountsSlashed(epochNumber: number): Promise<AccountSlashed[]> {
```

Comment:
be sure to add a patch changeset for contractkit about fixing LockeCeloWrapper. getAccountsSlashed for l2